modules/nvidia_plugin/README.md

```diff
@@ -171,7 +171,8 @@ The plugin supports the configuration parameters listed below. All parameters mu
 Parameter name  | Parameter values  | Default  | Description
 ------------- | ------------- | ------------- | -------------
 `NVIDIA_THROUGHPUT_STREAMS`   | `NVIDIA_THROUGHPUT_AUTO`, or non negative integer values  | 1  | Specifies number of CPU "execution" streams for the throughput mode. Upper bound for the number of inference requests that can be executed simultaneously.
-`NVIDIA_OPERATION_BENCHMARK`   | `NVIDIA_YES`, `NVIDIA_NO`  | `NVIDIA_NO`  | Specifies if operation level benchmark should be run for increasing performance of network
+`NVIDIA_OPERATION_BENCHMARK`   | `YES`, `NO`  | `NO`  | Specifies if operation level benchmark should be run for increasing performance of network
+`NVIDIA_USE_CUDA_GRAPH`   | `YES`, `NO`  | `YES`  | Specifies if NVIDIA Plugin should use CUDA graphs for performance acceleration
```

Comment:
Not sure that it good idea to expose new parameter, which turn on nothing. Users are not aware about planning features and it looks very strange.

Moreover, it is questionable whether we will need this parameter in future or we can turn on it always. Can we currently keep this parameter internal for development purposes only?

Reply:
Removed

Reply:
Moved `use_cuda_graph` property to `cuda_config.cpp` into `internal` namespace.